content/_changelogs/10.7.0.md

```diff
@@ -0,0 +1,39 @@
+- Adds [Svelte](https://svelte.dev/) component testing support. Addresses [#23431](https://github.com/cypress-io/cypress/issues/23431) and [#23432](https://github.com/cypress-io/cypress/issues/23432).
+- New informational banners will display instructions to assist with connecting projects to the Cypress Dashboard and recording a first test run. Users who don't plan to use the Cypress Dashboard can dismiss the banners to not see them in the future. Addresses [#22933](https://github.com/cypress-io/cypress/issues/22933), [#22934](https://github.com/cypress-io/cypress/issues/22934), [#22798](https://github.com/cypress-io/cypress/issues/22798), and [#22935](https://github.com/cypress-io/cypress/issues/22935)
+- Vue.js projects with a custom spec pattern configuration can now use the "Create Spec from Component" capability introduced in v10.5.0. Addresses [#23298](https://github.com/cypress-io/cypress/issues/23298)
+- Angular component testing projects can provide a custom configuration to the Webpack dev server using the `projectConfig` key. This enables support for projects that don't adhere to Angular CLI conventions. Addresses [#23161](https://github.com/cypress-io/cypress/issues/23161)
+- The Cypress `clock` can now be set to a new time using the `setSystemTime` function. Addresses [#15424](https://github.com/cypress-io/cypress/issues/15424)
+
+
+**Performance:**
+
+- The `cy.session()` command's behavior has been enhanced to no longer clear the dom after a validation function runs. This means a `cy.visit()` command is no longer required after running `cy.session()` when a validation function is used. Addresses [#22368](https://github.com/cypress-io/cypress/issues/22368).
+
+
+**Bugfixes:**
+
+- Fixed an issue where Cypress could crash if a websocket upgrade request was made over HTTPS. Fixes [#22217](https://github.com/cypress-io/cypress/issues/22217)
+- Fixed an issue where filtering with the `--spec` run mode option would not find any specs if the project was located at the root directory of a file system. Fixes [#23380](https://github.com/cypress-io/cypress/issues/23380)
+- Fixed an issue where projects using Angular v14.2.0 and above would be unable to run component tests. Fixes [#23585](https://github.com/cypress-io/cypress/issues/23585)
+- Fixed an issue where snapshots performed in XHR requests could reflect the primary domain instead of current domain. Fixes [#21496](https://github.com/cypress-io/cypress/issues/21496)
+- Introduced a new `GITHUB_RUN_ATTEMPT` environment variable when claiming specs from Cypress Cloud to address a potential scenario that would cause invalid Github Action requests to be accepted. Addressed in [#23445](https://github.com/cypress-io/cypress/pull/23445)
```

Comment:
What does "claiming specs from Cypress Cloud" mean? Is claiming maybe a typo? Or just lingo I'm unfamiliar with 😝

Reply:
🤷‍♂️ I was having trouble converting the User Facing Changelog entry from that PR into something more digestable. I'll ping Jaime in the morning to see if he can dumb it down for me